### PR TITLE
Add best practices guide

### PR DESCRIPTION
**Pull request by catherineluse**
This PR is intended to put this best practices guide https://github.com/yasker/longhorn/wiki/Best-practices-for-deploying-Longhorn-in-production into the Longhorn docs.

### CODE REVIEW

File: content/docs/0.8.1/best-practices.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+Since Longhorn doesn't currently support sharding between the different disks, we recommend using [LVM](https://en.wikipedia.org/wiki/Logical_Volume_Manager_(Linux)) to aggregate all the disks for Longhorn into a single partition, so it can be easily extended in the future.
+
+### Setting up Extra Disks
+
+Any extra disks must be written in the `/etc/fstab` directory to allow automatic mounting after the machine reboots.
+
+Don't use a symbolic link for the extra disks. Use `mount --bind` instead of `ln -s` and make sure it's in the `fstab` directory. For details, see [the section about multiple disk support.](../volumes-and-nodes/multidisk/#use-an-alternative-path-for-disk-on-the-node)
+
+## Configuring Defaults Before and After Installation
```
Comment (yasker) [inline content/docs/0.8.1/best-practices.md:54]: `Configuring Default Disks ...`

Comment (catherineluse) [reply content/docs/0.8.1/best-practices.md]: updated

**Comment by yasker** [comment]
LGTM.

**Comment by yasker** [comment]
@catherineluse I cannot find this in the latest staging. I remember we lost another change before. Is it possible something was wrong with the rebase?

**Pull request merged**